test/proposals/Addresses.sol

```diff
@@ -521,7 +521,7 @@ contract Addresses is Test, ChainIds {
         _addAddress(
             "JUMP_RATE_IRM_MOONWELL_WETH",
             8453,
-            0x4393277B02ef3cA293990A772B7160a8c76F2443
+            0x6bcE15B789e537f3abA3C60CB183F0E8737f05eC
```

Comment:
Let me know if I should add a new entry that overrides this one instead of editing in-place

Reply:
Editing in place seems fine here. What we'll do is just hold on merging the PR until the governance proposal succeeds, at which point the old IR model contract address is largely unnecessary to keep.